Find a functionally equivalent code:android.media.MediaFocusControl.dispatchMediaKeyEvent:COMMENT
Method Modifier: private     
Comment:/**
 * Handles the dispatching of the media button events to one of the registered listeners,
 * or if there was none, broadcast an ACTION_MEDIA_BUTTON intent to the rest of the system.
 * @param keyEvent a non-null KeyEvent whose key code is one of the supported media buttons
 * @param needWakeLock true if a PARTIAL_WAKE_LOCK needs to be held while this key event
 * is dispatched.
 */

@@ -4,17 +4,17 @@
     }
     Intent keyIntent = new Intent(Intent.ACTION_MEDIA_BUTTON, null);
     keyIntent.putExtra(Intent.EXTRA_KEY_EVENT, keyEvent);
-    synchronized (mRCStack) {
-        if (!mRCStack.empty()) {
+    synchronized (mPRStack) {
+        if (!mPRStack.empty()) {
             // send the intent that was registered by the client
             try {
-                mRCStack.peek().mMediaIntent.send(mContext, needWakeLock ? WAKELOCK_RELEASE_ON_FINISHED : 0, /*code*/
+                mPRStack.peek().getMediaButtonIntent().send(mContext, needWakeLock ? WAKELOCK_RELEASE_ON_FINISHED : 0, /*code*/
                 keyIntent, this, mEventHandler);
             } catch (CanceledException e) {
-                Log.e(TAG, "Error sending pending intent " + mRCStack.peek());
+                Log.e(TAG, "Error sending pending intent " + mPRStack.peek());
                 e.printStackTrace();
             }
         } else {

